lib/internal/module.js

```diff
@@ -83,43 +83,50 @@ const builtinLibs = [
   'string_decoder', 'tls', 'tty', 'url', 'util', 'v8', 'vm', 'zlib'
 ];
 
+// Make built-in modules available directly (loaded lazily).
 function addBuiltinLibsToObject(object) {
-  // Make built-in modules available directly (loaded lazily).
-  builtinLibs.forEach((name) => {
+  const configurable = true;
+  const enumerable = false;
```

Comment:
I don't think these are necessary. I would just keep these values inline to ensure no variable lookup has to be done by V8.